### PR TITLE
ADO-17883 Swap out icon link list on auth

### PR DESCRIPTION
**Pull request by ryan-reddekopp**
🎨

- create a new component. The button--ghost.
- increase version number

ADO LINK: https://dev.azure.com/AMA-Ent/AMA-Ent/_workitems/edit/17883

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [x] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [x] I've added new components to the style guide (or have a PR in to add them).
- [x] Any applicable version numbers have been updated.
- [x] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [x] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [x] I have written a detailed PR message explaining what is being changed and why
- [x] I’ve linked to any relevant VSTS tickets
- [x] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

File: app/assets/stylesheets/v3/layout_components/buttons/buttons.scss [hunk 1]
```diff
@@ -111,6 +111,44 @@
...
+    text-align: left;
+    align-items: center;
+    background-color: $white;
+    border: 3px solid $blue-main-brand;
+    min-height: 4.5rem;
+    padding: $base-padding/2 $base-padding*3 $base-padding/2 $base-padding;
+    color: $blue-main-brand;
+    vertical-align: middle;
+    line-height: 1.65rem;
+    position: relative;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/v3/layout_components/buttons/buttons.scss:126]: why do we need this?

File: app/assets/stylesheets/v3/layout_components/buttons/buttons.scss [hunk 2]
```diff
@@ -111,6 +111,44 @@
...
+    @extend .button;
+    display: flex;
+    text-align: left;
+    align-items: center;
+    background-color: $white;
+    border: 3px solid $blue-main-brand;
+    min-height: 4.5rem;
+    padding: $base-padding/2 $base-padding*3 $base-padding/2 $base-padding;
+    color: $blue-main-brand;
+    vertical-align: middle;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/v3/layout_components/buttons/buttons.scss:124]: I'm not sure you need this with align-items: center?

**Review by kaytcampbell — CHANGES_REQUESTED** [review]
A couple of other things:
I noticed in IE 11 they don't vertically center
image

Also, if the text spans multiple lines, it breaks the layout and the padding gets weird
image

File: app/assets/stylesheets/v3/layout_components/buttons/buttons.scss (hunk 1 above)
Comment (ryan-reddekopp) [reply app/assets/stylesheets/v3/layout_components/buttons/buttons.scss:126]: Oh no!! Maybe I won't be able to get away with flexbox after all. I'll look into a different solution.

Comment (ryan-reddekopp) [reply app/assets/stylesheets/v3/layout_components/buttons/buttons.scss:126]: We need this for the ::after position being set to absolute. Without explicitly setting the position of the parent, the arrows won't sit properly in each button.

File: app/assets/stylesheets/v3/layout_components/buttons/buttons.scss (hunk 2 above)
Comment (ryan-reddekopp) [reply app/assets/stylesheets/v3/layout_components/buttons/buttons.scss:124]: You are right! I forgot to remove it when I added flex to the buttons. Will fix!

File: app/assets/stylesheets/v3/layout_components/grids.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  &--3 {
+    @extend .grid;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  &--4 {
+    @extend .grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+}
```
Comment (kaytcampbell) [inline app/assets/stylesheets/v3/layout_components/grids.scss:23]: I see we created this file, but it isn't being used in your GK PR. Can we remove it?

**Review by kaytcampbell — APPROVED** [review]

**Pull request merged**